js/main.js

```diff
@@ -1,2 +1,162 @@
+		//Random number from 1 to 3
+		botChoice = Math.floor(Math.random() * 3) + 1
+
+		//If statement to assign numbers to string choices
+		if (botChoice === 1) {
+			botChoice = 'rock'
+		} else if (botChoice === 2) {
+			botChoice = 'scissors'
+		} else if (botChoice === 3) {
+			botChoice = 'paper'
+		} else {
+			console.log('Invalid choice.')
+		}
+	}
+
+	//Grabbing the clicked button id and converting that choice to a string
+	function assignUserChoice() {
+		userChoice = $(event.target).attr('id')
+	}
+
+	//Function to compare choices, declare a winner, and reward points
+	function determineWinner() {
+		if ( userChoice === botChoice ) {
+			$('#status').html(`<p>You chose <b><u>${userChoice}</u></b>. The bot chose <b><u>${botChoice}</u></b>.</p><p>Tie game!</p>`)
+		} else if ( userChoice === 'rock' && botChoice === 'paper') {
+			botWin()
+		} else if ( userChoice === 'rock' && botChoice === 'scissors') {
+			userWin()
+		} else if ( userChoice === 'paper' && botChoice === 'rock') {
+			userWin()
+		} else if ( userChoice === 'paper' && botChoice === 'scissors') {
+			botWin()
+		} else if ( userChoice === 'scissors' && botChoice === 'rock') {
+			botWin()
+		} else if ( userChoice === 'scissors' && botChoice === 'paper') {
+			userWin()
+		} else {
+			console.log('Error, invalid choices.')
+		}
+	}
+
+	//Assigning bonus points based on three wins
+	function bonusPoints() {
+		if ( botWins === 3) {
+			botWins = 0
+			botScore += 2
+			$('#computerScore').text(botScore)
+			$('#bonus-message').text('Bot gained 2 bonus points for 3 consecutive wins!')
+		} else if ( userWins === 3) {
+			userWins = 0
+			userScore += 2
+			$('#humanScore').text(userScore)
+			$('#bonus-message').text('You gained 2 bonus points for 3 consecutive wins!')
+		} else {
+			console.log('No bonus points awarded!')
+		}
+	}
+
+	function botWin() {
+		botScore += 1
+		botWins += 1
+		userWins = 0
+		$('#computerScore').text(botScore)
+		$('#status').html(`<p>You chose <b><u>${userChoice}</u></b>. The bot chose <b><u>${botChoice}</u></b>.</p><p>You lose!</p>`)
+	}
+
+	function userWin() {
+		userScore += 1
+		userWins += 1
+		botWins = 0
+		$('#humanScore').text(userScore)
+		$('#status').html(`<p>You chose <b><u>${userChoice}</u></b>. The bot chose <b><u>${botChoice}</u></b>.</p><p>You win!</p>`)
+	}
+
+	//Checks for first player to win 10 games and changes the game color
+	function roundOneWinner() {
+		if(botScore >= 10) {
+			$('#wise-message').text('Round 1 Winner: Bot')
+			$('body').css('color', 'yellow')
+			$('button').css('background', 'yellow')
+			$('.scoreboard').css('borderColor', 'yellow')
+			$('.scoreboard h2').css('borderBottomColor', 'yellow')
+		} else if(userScore >= 10) {
+			$('#wise-message').text('Round 1 Winner: You')
+			$('body').css('color', 'yellow')
+			$('button').css('background', 'yellow')
+			$('.scoreboard').css('borderColor', 'yellow')
+			$('.scoreboard h2').css('borderBottomColor', 'yellow')
+		}
+	}
```

Comment:
@cswebdesign Great job on breaking each logical portion into separate functions. To conform to the `DRY` principle and get the most out of the functions, consider passing arguments through so it can be extended in other parts of your code. This can be reused in the `roundTwoWinner` function, but there are certainly many ways to solve this. 

Great effort on the extra features!! 🥇 

check the solution for how things are being [compared](https://github.com/jsr20190826/rock-paper-scissors-solution/blob/master/js/main.js#L30)